01-Git_Bash_Make/bash_script

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
 mkdir guess_a_number
+cp main.c guess_a_number/
```

Comment:
@vometel,
I believe that according to the task, the temporary directory has to be created in `/tmp/` (otherwise you'r always responsible for cleaning up).

Also you should make sure that newly created temporary directory doesn't interfere with already available (e.g. from previous runs).